### PR TITLE
[refactor] core: use es6 rest parameter.

### PR DESCRIPTION
**Pull request by iyuq**
##### Checklist

- [x] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

File: lib/loader/egg_loader.js [hunk 1]
```diff
@@ -258,16 +258,15 @@ class EggLoader {
    * ```
    * @since 1.0.0
    */
-  loadFile(filepath) {
+  loadFile(filepath, ...args) {
```
Comment (popomore) [inline lib/loader/egg_loader.js]: Can you rename `args` to `inject`?

**Comment by iyuq** [comment]
好的，我是在这个基础上在提交一个commit就好了吗？

File: lib/loader/egg_loader.js (hunk 1 above)
Comment (iyuq) [inline lib/loader/egg_loader.js]: 好的，我是在这个基础上在提交一个commit就好了吗？

**Comment by iyuq** [comment]
@popomore  重命名过了。

**Pull request merged**